test/apiClients/AirtableApiClient.ts

```diff
@@ -1,5 +1,223 @@
+  email: 'info@arroyoseco.org',
+  phoneNumber: '(098) 765-4321',
+  url: 'https://www.arroyoseco.org/getinvolved.htm',
+  interestCategories: ['Water Organizations', 'Environmental Causes'],
+};
+
+const org1: Organization = {
+  name: 'Chrysalis Enterprises',
+  riverSection: 'Upper',
+  activity: 'Needs Volunteers',
+  description:
+    'Chrysalis is a nonprofit organization dedicated to creating a pathway to self-sufficiency for homeless and low-income individuals by providing the res...',
+  email: 'fakeemail@chrysalis.org',
+  phoneNumber: '(123) 456-7890',
+  url: 'https://changelives.org/',
+  interestCategories: ['Water Organizations'],
+};
+
+const org3: Organization = {
+  name: 'ctc',
+  riverSection: 'Lower',
+  activity: 'Needs Volunteers',
+  description: 'test data',
+  email: 'fakeemail@chrysalis.org',
+  phoneNumber: '(999) 999-9999',
+  url: 'https://changelives.org/',
+  interestCategories: ['Social Justice and Recreation'],
+};
+
+test('Returns one page of results properly', async t => {
+  const airtableApiClient: AirTableApiClient = new AirTableApiClient('keyzxy');
+
+  nock('https://api.airtable.com/v0/appEHr8iHguvEfXTQ')
+    .persist()
+    .log(data => console.log(data))
+    .get('/Organizations?view=Grid+view')
+    .reply(200, {
+      records,
+    });
+
+  const organizations: Organization[] = await airtableApiClient.getAllOrganizations();
+  const answer: Organization[] = [org1, org2, org3];
+  t.deepEqual(organizations, answer);
+});
+
+test('Returns one page of filtered river results properly', async t => {
+  const airtableApiClient: AirTableApiClient = new AirTableApiClient('keyzxy');
+
+  nock('https://api.airtable.com/v0/appEHr8iHguvEfXTQ')
+    .persist()
+    .log(data => console.log(data))
+    .get('/Organizations?view=Grid+view')
+    .reply(200, {
+      records,
+    });
+
+  const airTableFilters: AirTableFilters = {
+    riverSections: ['Middle'],
+  };
+
+  const organizations: Organization[] = await airtableApiClient.getAllOrganizations(airTableFilters);
+  const answer: Organization[] = [org2];
+  t.deepEqual(organizations, answer);
+});
+
+test('Returns one page of filtered interest categories results properly', async t => {
+  const airtableApiClient: AirTableApiClient = new AirTableApiClient('keyzxy');
+
+  nock('https://api.airtable.com/v0/appEHr8iHguvEfXTQ')
+    .persist()
+    .log(data => console.log(data))
+    .get('/Organizations?view=Grid+view')
+    .reply(200, {
+      records,
+    });
+
+  const airTableFilters: AirTableFilters = {
+    interestCategories: ['Environmental Causes'],
+  };
+
+  const organizations: Organization[] = await airtableApiClient.getAllOrganizations(airTableFilters);
+  const answer: Organization[] = [org2];
+  t.deepEqual(organizations, answer);
+});
+
+test('Returns one page of filtered both empty results properly', async t => {
```

Comment:
Run each of these tests with `test.serial`. Ava's tests run in parallel by default, and this can mess with mocking.
```suggestion
test.serial('Returns one page of filtered both empty results properly', async t => {
```